package.json

```diff
@@ -15,6 +15,7 @@
     "coverage:report:ci": "nyc report --reporter=lcovonly || true",
     "dev": "webpack serve",
     "dev:coverage": "npm start & cypress open --env COVERAGE=open",
+    "cypress:nocoverage": "cypress run --env COVERAGE=false",
```

Comment:
So now cypress isn't recording the run.. After looking at the orb I think we should probably rely on how it works and use the config rather than overriding the command: https://github.com/cypress-io/circleci-orb/blob/master/src/orb.yml#L1356